phys/module_shallowcu_driver.F

```diff
@@ -508,6 +571,68 @@ SUBROUTINE shallowcu_driver(                                       &
          CALL wrf_error_fatal('Lacking arguments for SHCU_NSCV in shallow cumulus driver')
       ENDIF
 
+  CASE (DENGSHCUSCHEME)
+      CALL wrf_debug(100,'in PSU-DENG Shallow Cu CPS')
+
+
+     IF ( bl_pbl_physics == 2 ) THEN   ! MYJ
+         DO j=jts,jte
+         DO i=its,ite
+         DO k=kts,kte
+            tke_scr(I,K,J) = tke_pbl(I,K,J)
+            kth_scr(I,K,J) = exch_h(I,K,J)
+            bbls_scr(I,K,J) = el_pbl(I,K,J)
+         ENDDO
+         ENDDO
+         ENDDO
+      ELSE IF ( bl_pbl_physics == 5 ) THEN   ! MYNN
+         DO j=jts,jte
+         DO i=its,ite
+         DO k=kts,kte
+            tke_scr(I,K,J) = 0.5 * qke(I,K,J)
+            kth_scr(I,K,J) = exch_h(I,K,J)
+            bbls_scr(I,K,J) = el_pbl(I,K,J)
+         ENDDO
+         ENDDO
+         ENDDO
+      ELSE
+         WRITE(message,*) 'PSU DENG ShCu currently does not support PBL option: ', &
+                            bl_pbl_physics
+         CALL wrf_error_fatal ( message )
```

Comment:
@dudhia Should this be checked in check_a_mundo? What about Boulac, QNSE, and MYNN 3.5, GBM schemes?

Reply:
@weiwangncar @dudhia 
Folks,
Let's make this an issue. There will be tweaking of the code, and check_a_mundo is among those modifications that can happen over the next several weeks.

Reply:
OK, we can deal with it later